hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/JsonMetadataRecordStoreMutationObserver.java

```diff
@@ -156,8 +159,9 @@ private Metadata initializeMetadata(Data key, Object value) {
                 return metadata;
             }
             return null;
-        } catch (Exception e) {
-            throw rethrow(e);
+        } catch (IOException e) {
```

Comment:
if we don't care `IOException`, can't we handle this inside the exception source? Seems `createSchema` is the source method throwing it, what about catching the exception there?

Reply:
I think ignoring the exception here makes more sense. The caller of metadata initializer should decide whether to allow invalid content.